packages/gestalt-codemods/_next/__testfixtures__/flex-gap-expansion.input.js

```diff
@@ -0,0 +1,48 @@
+// @flow strict
+import { Box, Flex, Text } from 'gestalt';
+
+const condition = true;
+const gapValue = 2;
+const alternateGapValue = 8;
+
+export default function TestComponent() {
+  return (
+    <Box>
+      <Flex gap={3}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+
+      <Flex direction="row" gap={3}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+
+      <Flex direction="column" gap={3}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+
+      <Flex gap={gapValue}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+
+      <Flex gap={condition ? 2 : 3}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+
+      <Flex gap={condition ? gapValue : alternateGapValue}>
+        <Text>Item 1</Text>
+        <Text>Item 2</Text>
+        <Text>Item 3</Text>
+      </Flex>
+    </Box>
```

Comment:
To be extra careful, is it worth repeating these with direction="column"?

Reply:
It's not really necessary…if you check out the codemod logic, the parts that determine what the new value is and where that new value goes are pretty independent. So it should be enough to test "does the new value go where expected given the `direction` value?" and "does the logic handle ternaries properly?" without needing to explicitly test every possible permutation.